pkg/images/defaults/useragent.go

```diff
@@ -0,0 +1,15 @@
+package defaults
```

Comment:
I do not think that `UserAgent` belongs to **image** defaults. In this case _image_ means container image. Please look for more appropriate package for this.

Reply:
Would `httputil` or `k8sutil` be more appropriate?

Reply:
I searched for `sigs.k8s.io/controller-runtime` in code (since config belongs to it) and found that it mostly is used in `operator/`. My first reaction would be `operator/pkg/utils`